theme/theme_test.go

```diff
@@ -20,3 +21,170 @@ func TestThemeChange(t *testing.T) {
 	fyne.CurrentApp().Settings().SetTheme(LightTheme())
 	assert.NotEqual(t, bg, BackgroundColor())
 }
+
+func TestTheme_Dark_ReturnsCorrectBackground(t *testing.T) {
+	fyne.CurrentApp().Settings().SetTheme(DarkTheme())
```

Comment:
Thinking more about this I don’t know that it’s right - we are asserting the theme colours through hard coded strings. Which means the cannot be changed without changing the tests too. Is there a more flexible approach?

Reply:
Only thing I can think is to create some variables to hold background color, similar to how we had the light background color before. Thoughts

Reply:
The question is what are we really testing? My best guess would be to SetTheme(DarkTheme()) then check the return is DarkTheme().BackgroundColor().
Adding more variables does not seem to add anything I don't think?

Reply:
good call! I didn't think to call DarkTheme() directly.  Fixed.